network-store-model/src/main/java/com/powsybl/network/store/model/Resource.java

```diff
@@ -36,6 +39,12 @@
     @ApiModelProperty("Resource attributes")
     private T attributes;
 
+    @JsonIgnore
```

Comment:
I find this very weird to have a reference to the client (as an UpdateResource) in the resource. Also this makes for weird behavior when you create a resource without a client it doesn't update, but with a client it does update.

There is an asymetry in the design between creation and update. Maybe making it more symmetric will lead to a more robust design ? For resource creation, everything related to the client happens in the network-store-client SwitchImpl (and other) classes, but for updates this happens in the network-store-model classes. I guess you did that to reduce verbosity, but maybe it's better to keep the code consistent between creation and update than having less verbose code.

Can you investigate on how verbose it becomes if update tracking happens at the network-store-client level ?